Add tests for getbinaries_kiwiproduct

eduardoj opened the pull request.

mlschroe commented:
You need to have some packages in the product, otherwise testing getbinaries_product is somewhat pointless

eduardoj commented:
> You need to have some packages in the product, otherwise testing getbinaries_product is somewhat pointless

I added a package. And I also fixed a problem I had with the tests changing `'srcserver' => $BSConfig::srcserver,` to `'srcserver' => 'http://srcserver',`. Please take another look.
